lib/paperclip.rb

```diff
@@ -56,7 +56,14 @@
 require 'paperclip/attachment_registry'
 require 'paperclip/filename_cleaner'
 require 'paperclip/rails_environment'
-require 'mime/types'
+
+begin
+  # Use mime/types/columnar if available, for reduced memory usage
+  require 'mime/types/columnar'
+rescue LoadError
+  require 'mime/types'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.